go.mod

```diff
@@ -42,6 +43,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.11.0
 )
 
+require github.com/google/go-cmp v0.5.6
```

Comment:
Small nitpick. This looks odd to me, was this manually added or through `go get github.com/google/go-cmp@v0.5.6` ?

Reply:
came up after I ran `go mod tidy && go mod vendor && go mod verify` . Should I manually move it under one of the require?

Reply:
Nah, that's fine, was just curious if the go tool created it